docs/content/utilities/layout.md

```diff
@@ -118,6 +118,11 @@ Overflow utilities can also target x- and y-axes independently via:
 * `.overflow-y-scroll`
 * `.overflow-y-visible`
 
+### Overflow floats
+Overflow utilities can be applied or changed per [breakpoint](/objects/grid#breakpoints). 
+
+Each responsive overflow utility is applied to the specified breakpoint and up, using the following formula:  `overflow-[breakpoint]-[axis]-[property]`. For example: `overflow-x-scroll` or `overflow-md-x-visible`.
```

Comment:
Maybe to avoid having an extra nav item show up in the sidebar, we could only add an extra paragraph. And maybe make the word "responsive" bold:

```suggestion
Overflow utilities can be applied or changed per [breakpoint](/objects/grid#breakpoints). Each **responsive** overflow utility is applied to the specified breakpoint and up, using the following formula:  `overflow-[breakpoint]-[axis]-[property]`. For example: `overflow-x-scroll` or `overflow-md-x-visible`.
```